Tests/Contour.RabbitMq.Tests/BasicRequestResponseSpecs.cs

```diff
@@ -557,6 +557,69 @@ public void should_receive_successfully()
             }
         }
 
+        [TestFixture]
+        [Category("Integration")]
+        public class WhileProcessingIncomingRequest : RabbitMqFixture
+        {
+            /// <summary>
+            /// Ответы должны приходить.
+            /// </summary>
+            [Test]
+            public void ShouldSuccessfullyMakeConcurrentRequestsToSingleRemoteConsumer()
+            {
+                IBus producer = this.StartBus(
+                    "producer",
+                    cfg =>
+                    {
+                        cfg.Route("originalRequest")
+                            .WithDefaultCallbackEndpoint();
+                    });
+
+                var requestingConsumer = this.StartBus(
```

Comment:
Why is there an additional consumer? The headers should be updated with a new Id on each producer request made in parallel and each producer should receive its own response.